server/views/partials/caseLicenceConditions.njk

```diff
@@ -0,0 +1,88 @@
+<h1 class='govuk-heading-m govuk-visually-hidden'>Licence conditions</h1>
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-full">
+
+        {% for offence in caseSummary.offences %}
+            <div class='govuk-!-margin-bottom-8'>
+                <h2 class='govuk-heading-m'>Licence {{ loop.index }}</h2>
+                {% set offenceHtml %}
+                    <ul class='govuk-list'>
+                        {% for offenceItem in offence.offences %}
+                            <li>{{ offenceItem.description }}</li>
+                        {% endfor %}
+                    </ul>
+                {% endset %}
+                {{ govukSummaryList({
+                    classes: 'govuk-summary-list--no-border',
+                    rows: [
+                        {
+                            key: { html: 'Offence' },
+                            value: { html: offenceHtml }
+                        },
+                        {
+                            key: { text: 'Sentence description' },
+                            value: { text: offence.sentenceDescription }
+                        },
+                        {
+                            key: { text: 'Sentence length' },
+                            value: { text: offence.sentenceOriginalLength + ' ' + offence.sentenceOriginalLengthUnits }
+                        },
+                        {
+                            key: { text: 'Sentence start date' },
+                            value: { text: formatDateTimeFromIsoString({isoDate: offence.sentenceStartDate}) }
+                        },
+                        {
+                            key: { text: 'Release from custody date' },
+                            value: { text: formatDateTimeFromIsoString({isoDate: caseSummary.releaseSummary.lastRelease.date}) }
```

Comment:
Also, did we want to display the 'caseSummary.releaseSummary.lastRecall.date' somewhere?